### DISCUSSION TITLE
Sync error (Onedrive personal)

### DISCUSSION THREAD

**Discussion by goodknight911** [remotely-save/remotely-save#83] [Q&A] [answered]
Thanks for this great sync plugin. I have been using this plugin smoothly for months. But these days it always showed error info as below. Appreciate for any feedback or help. 

![image](https://user-images.githubusercontent.com/48895526/163699115-b70e95d9-6e67-4ace-a073-f30d67c14eac.png)

**Comment by fyears** [comment, marked as answer]
There was a bug in the old android version of plugin.

1. please firstly make a full backup of local files to avoid bad things
2. please go to onedrive website, and manually check if the note files were corrupted or not
3. if some notes or files were corrupted, you could consider directly delete them, sorry…
4. please delete the _remotely-save-metadata-on-remote.json file. you met the error because of this file being corrupted. it’s usually safe to directly delete it.
5. please update the plugin to latest version, and start the sync again

**Reply by goodknight911** [reply]
Thanks so much. Just delete _remotely-save-metadata-on-remote.json from onedrive, then everything is fine now.  👍
